ls1intum/Artemis
# `Iris`: Get offered models from Pyris

### Checklist
#### General
#### Server

### Motivation, Context and Description
Since introducing a health endpoint to Pyris, we also want to provide a dynamic model list provided by Pyris.
Available models can be fetched through a newly created endpoint.
It is connected to ls1intum/Pyris#12

### Steps for Testing
Go to the global/course/exercise settings and observe the new dropdown, which fetches its data from Pyris. (Only works if ls1intum/Pyris#12 is deployed as well)

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots
![grafik](https://github.com/ls1intum/Artemis/assets/18218285/c1060c22-4362-4eff-9df2-9ee25dacf874)
